website/docusaurus.config.js

```diff
@@ -29,18 +29,27 @@ module.exports = {
                         {
                             to: 'docs',
                             label: 'Docs',
-                            activeBaseRegex: 'client-js/docs(?!.*/changelog)',
+                            activeBaseRegex: '/docs(?!/changelog|/examples)',
+                        },
+                        {
+                            to: 'docs/examples',
+                            label: 'Examples',
+                            activeBaseRegex: '/docs/examples',
                         },
                         {
                             to: 'api',
-                            label: 'API',
-                            activeBaseRegex: 'client-js/api',
+                            label: 'Reference',
+                            activeBaseRegex: '/api',
```

Comment:
are you sure this is correct?

FYI we also want to change the URL to `reference` in those API links in project sites

Reply:
![obrazek](https://user-images.githubusercontent.com/61918049/216300097-770b3b6c-7f85-4d73-bf29-1e30abb780b4.png)
I'm 95% sure, does something look off to you? 

Also, the [`plugin-typedoc-api`](https://www.npmjs.com/package/docusaurus-plugin-typedoc-api) plugin we're using is specifically registering the `/api` route with no override option (that is, if we don't want to `patch-package` some more).

Reply:
> Also, the [plugin-typedoc-api](https://www.npmjs.com/package/docusaurus-plugin-typedoc-api) plugin we're using is specifically registering the /api route with no override option (that is, if we don't want to patch-package some more).

Are you sure about that? There should be `routeBasePath` option, it's just a plugin, like the docs, I believe you can control this on docusaurus level.

Reply:
right, but there is the conflict, so please rebase :]

Reply:
> There should be `routeBasePath` option
yup, my bad, just got this working 👍️